MvvmCross.Android.Support/V7.AppCompat/MvxAppCompatSetup.cs

```diff
@@ -50,4 +46,15 @@ protected override void FillBindingNames(IMvxBindingNameRegistry registry)
             base.FillBindingNames(registry);
         }
     }
+
+    public abstract class MvxAppCompatSetup<TApplication> : MvxAppCompatSetup
+        where TApplication : IMvxApplication, new()
+    {
+        protected override IMvxApplication CreateApp() => Mvx.IocConstruct<TApplication>();
```

Comment:
@nickrandolph are you sure this will work? At first glance looks like [this](https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/Mvx.cs#L157) won't be ready at this point

Reply:
This ran up ok for me on Android - this is late executed, so the IoC container has initialized by the time this is called.